spec/acceptance/vhost_spec.rb

```diff
@@ -964,7 +964,7 @@ class { 'apache': }
     describe file($ports_file) do
       it { is_expected.to be_file }
       if fact('osfamily') == 'RedHat' && fact('operatingsystemmajrelease') == '7' ||
-         fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') =~ %r{(14\.04|16\.04)} ||
+         fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') =~ %r{(14\.04|16\.04|18\.04)} ||
```

Comment:
I think this is all supported Ubuntu versions.

Reply:
fixed